src/components/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Tag from '@/components/Tag';
+import { Meta } from '@storybook/react';
+import mdx from './Tag.stories.mdx';
+
+export default {
+  title: 'Components/Tag',
+  component: Tag,
+  parameters: {
+    docs: {
+      page: mdx
+    }
+  }
+} as Meta;
+
+const Template = args => {
+  return <Tag {...args} />;
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  children: 'Testing Tag'
+};
```

Comment:
@raphaelsattler can we consider an example where the user passes in a JSX/HTML element? Are we expecting this for a first version of the component?